gcp-application-cloud-run/iam.tf

```diff
@@ -0,0 +1,8 @@
+resource "google_cloud_run_service_iam_member" "public-access" {
```

Comment:
What is this role for?

Reply:
This is so the cloud run instance can be invoked by anonymous (public) users.

Reply:
Cool